### PR TITLE
Show correct wallet to add token to, in Transaction modal

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
After submitting a tx, there's a tx modal poping up and there's a button to add the token to the wallet.
Before, it was fixed to Metamask, even if Coin98 or Brave is being used
After, wallet detection is added. Only injected wallets will have that button. If it's the injected wallet used by TrustWallet in-app browser, the button will be 'Add token to **your wallet**'

Before:
image

### CODE REVIEW

File: src/components/TransactionConfirmationModal/index.tsx [hunk 1]
```diff
@@ -44,6 +45,40 @@ const StyledLogo = styled.img`
   width: 16px;
   margin-left: 6px;
 `
+
+const getBrowserWalletConfig = () => {
+  const { ethereum } = window
+  const hasInjectedWallet = !!ethereum
+  const isCoin98 = hasInjectedWallet && ethereum.isCoin98
+  const isBraveWallet = hasInjectedWallet && ethereum.isBraveWallet
+  const isMetaMask = hasInjectedWallet && ethereum.isMetaMask
```
Comment (viet-nv) [inline src/components/TransactionConfirmationModal/index.tsx:54]: `const {isCoin98, isBraveWallet, isMetaMask} = ethereum || {}`

Comment (viet-nv) [reply src/components/TransactionConfirmationModal/index.tsx:54]: my style

Comment (hungdoansy) [reply src/components/TransactionConfirmationModal/index.tsx:54]: Yeah, make sense

Comment (hungdoansy) [reply src/components/TransactionConfirmationModal/index.tsx]: updated

**Review by viet-nv — APPROVED** [review]

**Pull request merged**